package.json

```diff
@@ -31,23 +31,23 @@
   },
   "dependencies": {
     "@paulcbetts/system-idle-time": "^1.0.4",
-    "electron-updater": "3.0.4",
+    "electron-updater": "4.0.4",
     "electron-windows-notifications": "^3.0.1",
     "freedesktop-notifications": "^1.2.3",
-    "fs-jetpack": "^2.1.1",
-    "lodash": "^4.17.10",
+    "fs-jetpack": "^2.2.0",
+    "lodash": "^4.17.11",
     "spellchecker": "^3.5.0",
     "tmp": "^0.0.33"
   },
   "optionalDependencies": {},
   "devDependencies": {
     "@rocket.chat/eslint-config": "^0.1.2",
-    "chai": "^4.1.2",
-    "conventional-changelog-cli": "^2.0.5",
-    "electron": "^2.0.10",
-    "electron-builder": "^20.28.4",
+    "chai": "^4.2.0",
+    "conventional-changelog-cli": "^2.0.11",
+    "electron": "^3.0.9",
```

Comment:
This introduce many breaking changes: https://github.com/electron/electron/pull/13050